sql/api/src/main/scala/org/apache/spark/sql/types/StructField.scala

```diff
@@ -147,6 +147,17 @@ case class StructField(
     if (metadata.contains("comment")) Option(metadata.getString("comment")) else None
   }
 
+  /**
+   * Return the default value of this StructField.
```

Comment:
we should add more comments here, to highlight the difference between this and the current/existing default values.